forui/lib/src/widgets/button/button.dart

```diff
@@ -18,34 +18,70 @@ part 'button_content_style.dart';
 
 /// A button.
 class FButton extends StatelessWidget {
-  /// The style.
+
+  /// The design. Defaults to [FBadgeVariant.primary].
   final FButtonDesign design;
 
-  /// Called when the FButton is tapped or otherwise activated.
-  final VoidCallback? onPressed;
+  /// A callback for when the button is pressed.
+  final VoidCallback? onPress;
+
+  /// A callback for when the button is long pressed.
+  final VoidCallback? onLongPress;
+
+  /// True if this widget will be selected as the initial focus when no other node in its scope is currently focused.
+  ///
+  /// Ideally, there is only one widget with autofocus set in each FocusScope. If there is more than one widget with
+  /// autofocus set, then the first one added to the tree will get focus.
+  ///
+  /// Defaults to false.
+  final bool autofocus;
+
+  /// An optional focus node to use as the focus node for this widget.
+  ///
+  /// If one is not supplied, then one will be automatically allocated, owned, and managed by this widget. The widget
+  /// will be focusable even if a [focusNode] is not supplied. If supplied, the given `focusNode` will be hosted by this
+  /// widget, but not owned. See [FocusNode] for more information on what being hosted and/or owned implies.
+  ///
+  /// Supplying a focus node is sometimes useful if an ancestor to this widget wants to control when this widget has the
+  /// focus. The owner will be responsible for calling [FocusNode.dispose] on the focus node when it is done with it,
+  /// but this widget will attach/detach and reparent the node when needed.
+  final FocusNode? focusNode;
+
+  /// Handler called when the focus changes.
+  ///
+  /// Called with true if this widget's node gains focus, and false if it loses focus.
+  final ValueChanged<bool>? onFocusChange;
 
   /// The builder.
   final Widget Function(BuildContext, FButtonStyle) builder;
 
-  /// Creates a [FButton] widget.
+  /// Creates a [FButton].
   FButton({
+    required this.onPress,
```

Comment:
I don't think `onPress` should be required.

Reply:
I think it should be, it's required for buttons and switches in material & cuptertino.